app/driver/onewire.c

```diff
@@ -180,7 +180,6 @@ void onewire_write(uint8_t pin, uint8_t v, uint8_t power /* = 0 */) {
   if ( !power) {
```

Comment:
If you look at the code for DIRECT_MODE_INPUT in the ROM, then you will see that it already masks interrupts during this operation so the noInterrupts / interrupts are redundant.